src/app/meetups/meetups.component.ts

```diff
@@ -67,6 +94,55 @@ export class MeetupsComponent implements OnInit, AfterViewInit {
     };
   }
 
+  deleteMeetups(meetups) {
+    // Deletes multiple meetups
+    return () => {
+      const deleteMeetupArr = meetups.map((meetup) => {
+        return { _id: meetup._id, _rev: meetup._rev, _deleted: true };
+      });
+      this.couchService.post(this.dbName + '/_bulk_docs', { docs: deleteMeetupArr })
+        .subscribe((data) => {
+          this.getMeetups();
+          this.selection.clear();

```

Comment:
Move this line into getMeetups, so it clears the selection whenever we reset the data array.